config.sample.json

```diff
@@ -11,5 +11,6 @@
             "matrix.org"
         ]
     },
-    "welcomeUserId": "@RiotBot:matrix.org"
+    "welcomeUserId": "@RiotBot:matrix.org",
+    "welcomePageUrl": "https://about.riot.im"
```

Comment:
Oh I see, home.html is the default if one isn't defined. I think this is going to provide quite strange behaviour for people who just copy the sample config assuming the values in it are sane (ie. a lot of people probably)